scala-api/src/main/scala/com/cloudera/livy/scalaapi/LivyScalaClient.scala

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to Cloudera, Inc. under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Cloudera, Inc. licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cloudera.livy.scalaapi
+
+import java.io.File
+import java.net.URI
+import java.util.concurrent.{ScheduledFuture, ScheduledThreadPoolExecutor, TimeUnit, Future => JFuture}
+
+import com.cloudera.livy._
```

Comment:
Did you run "mvn install" or "mvn verify"? I still see style issues that those commands would have flagged.